### DISCUSSION TITLE
如何切换选中菜单item 的icon 样式?

### DISCUSSION THREAD

**Discussion by xuoutput** [ant-design/ant-design#39578] [Q&A]
使用 antd 的图标, 在菜单被选中的时候, 会变蓝色, 如下图的 navigation one.
<img width="837" alt="image" src="https://user-images.githubusercontent.com/16385443/207793364-4487cb20-8548-4f52-b873-2f0bb1797851.png">

<img width="840" alt="image" src="https://user-images.githubusercontent.com/16385443/207793329-49e096ec-24b6-4100-b7a0-953cf5c96453.png">

相关代码也很简单

```
const items: MenuProps['items'] = [
  {
    label: 'Navigation One',
    key: 'mail',
    icon: <MailOutlined />,
  },
]
```

 但如果使用的是自定义图片, 有什么简单的办法来做到相同的功能呢?

**Comment by zombieJ** [comment]
`fill="currentColor"`

**Reply by xuoutput** [reply]
大佬, 没用 webpack, 用了 vite, 是不是 自定义 SVG 图标 功能就不能用了
<img width="675" alt="image" src="https://user-images.githubusercontent.com/16385443/207812715-f5062710-06f5-47e2-8071-9d917e30c218.png">

**Comment by zombieJ** [comment]
这个是 svg 的属性，需要 svg 自己加上这个

**Reply by xuoutput** [reply]
https://codesandbox.io/s/ding-bu-dao-hang-antd-5-0-7-forked-62ky4e?file=/demo.tsx
<img width="1134" alt="image" src="https://user-images.githubusercontent.com/16385443/208070879-a86a7885-69ce-46a5-b97f-dc6840b20268.png">
好像不行诶

**Reply by xuoutput** [reply]
这边是我写错 svg 了
